src/hooks/proposal/useProposalsCoreData.ts

```diff
@@ -22,46 +25,91 @@ export function useProposalsCoreData(filter: ProposalsFilter) {
     let unmounted = false;
 
     setLoading(true);
-    api()
-      .getProposalsCore({
-        filter: {
-          callId,
-          instrumentId,
-          proposalStatusId,
-          questionaryIds,
-          questionFilter: questionFilter && {
-            ...questionFilter,
-            value:
-              JSON.stringify({ value: questionFilter?.value }) ?? undefined,
-          }, // We wrap the value in JSON formatted string, because GraphQL can not handle UnionType input
-          text,
-        },
-      })
-      .then((data) => {
-        if (unmounted) {
-          return;
-        }
 
-        if (data.proposalsView) {
-          setProposalsData(
-            data.proposalsView.map((proposal) => {
-              return {
-                ...proposal,
-                status: proposal.submitted ? 'Submitted' : 'Open',
-                technicalStatus: getTranslation(
-                  proposal.technicalStatus as ResourceId
-                ),
-                finalStatus: getTranslation(proposal.finalStatus as ResourceId),
-              } as ProposalViewData;
-            })
-          );
-        }
-        setLoading(false);
+    if (currentRole === UserRole.INSTRUMENT_SCIENTIST) {
+      api()
+        .getInstrumentScientistProposals({
+          filter: {
+            callId,
+            instrumentId,
+            proposalStatusId,
+            questionaryIds,
+            questionFilter: questionFilter && {
+              ...questionFilter,
+              value:
+                JSON.stringify({ value: questionFilter?.value }) ?? undefined,
+            },
+            text,
+          },
+        })
+        .then((data) => {
+          if (unmounted) {
+            return;
+          }
+          console.log(data);
```

Comment:
I don't think we should log all raw proposal data to the console

Reply:
Sorry missed that